src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -840,7 +853,7 @@ def resume_privacy_request(
         privacy_request_id=privacy_request.id,
         from_webhook_id=webhook.id,
     )
-    return privacy_request
+    return privacy_request  # type: ignore[return-value]
```

Comment:
I've chosen to ignore these as it's a common pattern where by returning a `PrivacyRequest` it coerces to a `PrivacyRequestResponse`